packages/fxa-settings/src/components/LinkedAccounts/en-US.ftl

```diff
@@ -7,4 +7,5 @@ la-unlink-account-button = Unlink
 la-unlink-heading = Unlink from third party account
 la-unlink-content = Are you sure you want to unlink your { -brand-google } account? Unlinking your account does not automatically sign you out of those services. To do that, you will need to manually sign out from the Connected Services section.
 la-unlink-content-2 = Are you sure you want to unlink your account? Unlinking your account does not automatically sign you out of those services. To do that, you will need to manually sign out from the Connected services section.
```

Comment:
We can remove `-2` and `-1` (L8 and L9) since we're not using those anymore, which is what we typically do to not bloat the generated file with strings we're no longer using. I think that helps give a more accurate number on what remains to be localized percentage wise across languages too too.